packages/www/components/Dashboard/StatusBadge/index.tsx

```diff
@@ -48,10 +50,19 @@ const StatusBadge = ({
       typeof timestamp === "string" ? Date.parse(timestamp) : timestamp;
     return !tsNum ? moment.invalid() : moment.unix(tsNum / 1000);
   }, [timestamp]);
-  if (!timeAgo.isValid() || style.noTooltip) {
+  if (
+    (!timeAgo.isValid() && !tooltipText) ||
+    (style.noTooltip && !tooltipText)
+  ) {
     return badge;
   }
-  return <Tooltip content={timeAgo.fromNow()}>{badge}</Tooltip>;
+  let contentText: string;
+  if (tooltipText && timestamp) {
```

Comment:
Gotta check if timeAgo is valid instead

```suggestion
  if (tooltipText && timeAgo.isValid()) {
```